src/components/Chat/InputMessageV2.tsx

```diff
@@ -170,6 +370,11 @@ const InputMessageV2 = ({
     );
   };
 
+  const plusButtonStyle = React.useCallback(() => {
+    if (type === SIGNED) return colors.darkBlue;
+    return colors.bondi_blue;
+  }, []);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `plusButtonStyle` function is using a hardcoded string comparison to determine the button color. This could lead to potential bugs if the `type` value changes in the future. Consider using constants for these comparisons.

```diff
-  const plusButtonStyle = React.useCallback(() => {
-    if (type === 'SIGNED') return colors.darkBlue;
-    return colors.bondi_blue;
-  }, []);
+  const plusButtonStyle = React.useCallback(() => {
+    if (type === SIGNED) return colors.darkBlue;
+    return colors.bondi_blue;
+  }, []);
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->